otterai/llm_client.py

```diff
@@ -0,0 +1,21 @@
+import os
+import httpx
+from langchain_openai import ChatOpenAI
+
+class LLMClient:
+    _instance = None
+
+    def __new__(cls) -> 'LLMClient':
+        if cls._instance is None:
+            cls._instance = super(LLMClient, cls).__new__(cls)
+            cls._instance.client = ChatOpenAI(
+                model_name=os.getenv('INPUT_MODEL', 'gpt-4-turbo-preview'),
```

Comment:
🔑 Consider adding error handling for the 'os.getenv' call to handle cases where 'INPUT_MODEL' is not set.